qhub/template/image/jupyterlab/environment.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
  - retrolab
 
  # jupyterlab extensions
-#  - dask_labextension >=5.0.0
+ - dask_labextension == 5.3.0
```

Comment:
Can we make this a forward pin to `>=5.3.0`?

Reply:
no problem, I just updated because the yarn version in there was breaking the ci